build.sbt

```diff
@@ -33,7 +33,7 @@ import com.typesafe.sbt.packager.docker._
 dockerPermissionStrategy := DockerPermissionStrategy.Run
 dockerVersion := Some(DockerVersion(18, 9, 0, Some("ce")))
 dockerRepository := sys.env.get("DOCKER_REPOSITORY")
-dockerBaseImage := "adoptopenjdk/openjdk12:x86_64-alpine-jre-12.33"
+dockerBaseImage := "openjdk:12-alpine"
```

Comment:
Oh.  They actually made a JDK 12 branch available.  IIRC, that's why we went with adoptopenjdk in the first place.

Reply:
yeah! :) We actually used it for the bias-correct service since it had libraries that supported Kamon, but the image is almost twice the size of the adoptopenjdk so I hesitated to use it here.